src/v2/util/Animations.js

```diff
@@ -0,0 +1,146 @@
+/*!
```

Comment:
What is changing from the existing file, `src/util/Animations.js` for V2?

Reply:
Nothing is changing in `Animations .js` for V2. This is just a dependency that had to be ported over with `BaseLoginRouter.js`.

Reply:
Would it work to do `../../util/Animations.js` ? Or perhaps a small refactor to include a `/shared` folder alongside `/v2` for these types of shared, common classes. When its a mono repo, shared can be its own module.

Reply:
Yeah, thats my next step. I am still trying to find out and see if we really need these in V2 world, or if there is room to delete these files. If not I will move them to a shared folder.